canvas_modules/common-canvas/src/common-properties/form/ControlInfo.js

```diff
@@ -184,6 +186,6 @@ export class Control {
 		if (settings.buttons) {
 			this.buttons = settings.buttons;
 		}
-		this.light = settings.light || false;
+		this.light = get(settings, "light", true);
```

Comment:
@nabilIbm Was there a reason for this change? Its not related to the `trimSpaces` work